web/crux-ui/src/components/products/product-versions-section.tsx

```diff
@@ -17,18 +18,21 @@ const ProductVersionsSection = (props: ProductVersionsSectionProps) => {
 
   const onClick = (versionId: string) => router.push(versionUrl(props.productId, versionId))
 
-  return (
+  return props.versions ? (
```

Comment:
we shouldn't bury deep down null checks.
please set versions to non-nullable, and simply check for null at the place of usage